diff --git a/scripts/lib/wic/filemap.py b/scripts/lib/wic/filemap.py
index abbf958b8c..7edf224291 100644
--- a/scripts/lib/wic/filemap.py
+++ b/scripts/lib/wic/filemap.py
@@ -37,8 +37,9 @@ def get_block_size(file_obj):
     """
     # Get the block size of the host file-system for the image file by calling
     # the FIGETBSZ ioctl (number 2).
-    binary_data = fcntl.ioctl(file_obj, 2, struct.pack('I', 0))
-    bsize = struct.unpack('I', binary_data)[0]
+#    binary_data = fcntl.ioctl(file_obj, 2, struct.pack('I', 0))
+#    bsize = struct.unpack('I', binary_data)[0]
+    bsize = 0
     if not bsize:
         import os
         stat = os.fstat(file_obj.fileno())
